projects/demo/src/modules/markup/spaces/spaces.module.ts

```diff
@@ -1,14 +1,17 @@
+import { ClipboardModule } from '@angular/cdk/clipboard';
```

Comment:
That's weird. Prettier should have had removed the spaces on precommit 🤔

Reply:
@fynnfeldpausch what IDE do you use?

Reply:
VSC 🤔

Reply:
Did you need to delay something like 1-2 seconds making a commit or was that instantly?